### ISSUE TITLE
Contribution to the repository 

### ISSUE THREAD

**Issue by NikosNtigas** [ankiedos/Console-Game-Engine#2]
I am doing a university projects where they encourage us to contribute to open source projects. So I am asking if it's ok to sending pull requests. I will like you to inform me about your thinking in this Game library.  A simple discussion about new features and ideas that might be useful, helpful. In order for me to help you as much as possible. 
Thx for your time.

**Comment by ankiedos** [comment]
If I understood correctly, you want to help me with this project, don't you?
And yes, it's ok. I'm sure your help will be very valuable.

**Comment by ankiedos** [comment]
Would you like me to mark you as contributor?

**Comment by NikosNtigas** [comment]
I don't have any problem with that, whatever you want, you can cancel always merge my pull request. 
Also I was thinking about moving the player language into the namespace game as game language.

**Comment by ankiedos** [comment]
It's interesting idea, but I think that it is better for `Player::` ns, because it is player's attribute.
I will think about it and answer you tomorrow. Do I translate it well? "Echeis chasei" <-> "You lost".

**Comment by NikosNtigas** [comment]
I saw that and No.
Do not worry it I will translate it correctly.

**Comment by ankiedos** [comment]
Thanks. Google Translate doesn't always translate well.

**Comment by NikosNtigas** [comment]
To be more specific it is actually pronation translated correctly, the problem is that is written in Latin alphabet not in Greek. In Greece we call it Greeklish, Greek pronation with Latin alphabet.

**Comment by ankiedos** [comment]
Oh, sorry... RODO is abbreviation of Polish name for EU Regulation on the Protection of Personal Data.

**Comment by ankiedos** [comment]
If I offended you with this, I can edit description of the latest pre-release and files within this tag to remove your nickname.

Changing the subject, what you think about creating library-specific integration with JSON using `nlohmann::json` struct? There would be such things as `game_lang` and player attributes, maybe we add something else?

**Comment by ankiedos** [comment]
I changed repository's name, luckily GitHub sends emails about new comments etc., so you can see this.
Now this repo's name is Console-Game-Engine.

**Comment by NikosNtigas** [comment]
Wait what? no I did not got offended. I just have a lot on going project for university and mine that's all sorry for not replying sooner. I do not care at all about showing my real name or nickname, my real name in already in my github profile so three is not point getting mad. I would like to help with repo but I will not do the university project that I told you before. So... that's why I did not reply sooner.
